keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -439,8 +439,26 @@ apiserver network proxy do not have to be stopped on rollback.
 All capabilities from in-tree cloud providers will be re-disabled.
 
 ###### Are there any tests for feature enablement/disablement?
```

Comment:
Also mention here that we've updated all existing CI jobs to now run with external providers. This means every (or at least "most") e2e tests can pass on a cluster with in-tree cloud provider disabled. 

Some testgrid links to reference:
* https://testgrid.k8s.io/provider-gcp-periodics#disable-cloud-provider -- this was the first periodic job that ran with cloud provider disabled which we had for several releases now)
* https://testgrid.k8s.io/provider-gcp-periodics#E2E%20Full%20-%20Cloud%20Provider%20GCP%20-%20with%20latest%20k8s.io/kubernetes -- this ensures any breaking changes to k8s.io/cloud-provider are caught early 
* https://testgrid.k8s.io/google-gce#pull-kubernetes-e2e-gce -- example presubmit required for every PR that runs with in-tree cloud providers disabled

Reply:
These are rather feature test then enablement/disablement, but I agree that helps too.

That being said, I sugggest switching this answer to something like:

We have a huge number of feature tests, including;
- unit tests under https://github.com/kubernetes/cloud-provider/blob/release-1.28
- e2e test suites running with in-tree providers disabled:
    https://testgrid.k8s.io/provider-gcp-periodics#disable-cloud-provider
    https://testgrid.k8s.io/provider-gcp-periodics#E2E%20Full%20-%20Cloud%20Provider%20GCP%20-%20with%20latest%20k8s.io/kubernetes

For enablement/disablement tests, we are relying on manul tests that were done downstream on at least two large cloud providers: GCP and AWS.

Reply:
thanks! updating